docs/fastdds/transport/zero_copy/zero_copy.rst

```diff
@@ -0,0 +1,161 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+.. include:: ../../../03-exports/roles.include
+
+.. _transport_zero_copy:
+
+Zero-Copy
+=========
+
+This section explains the Zero-Copy transfer mode implemented in *Fast DDS*.
+The Zero-Copy transfer allows the transmission of data between DDS |DataWriter| and |DataReader| entities without
+making copies of the data being transmitted.
+These samples are stored in a shared memory region, which is accessible by multiple processes implementing an
+application using Fast DDS. In other words, the DataWriter has its History in a memory mapped file accessible to
+the DataReader.
+
+The figure below shows a comparison between the different transports available in Fast DDS.
+
+.. figure:: /01-figures/fast_dds/transport/transport_comparison.png
+    :align: center
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
```

Comment:
This is a great intro for DataSharing and I'm going to rip it off shamelessly. 
In the description for zero-copy I'd say that it uses data-sharing transfer between entities and a loan mechanism of data buffers in order to avoid copying the data.

Probably moving the Overview section (where loans are mentioned) to the intro makes sense

Reply:
:+1: 
Check after Shared Memory is added.